pyaedt/edb_core/net_class.py

```diff
@@ -212,3 +282,44 @@ def create(self, name, net_p, net_n):
         diff_pair.api_create(name)._api_set_differential_pair(net_p, net_n)
 
         return self.items[name]
+
+    @pyaedt_function_handler
+    def auto_identify(self, positive_differentiator="_P", negative_differentiator="_N"):
+        """Generate diffperential pairs.
```

Comment:
This needs to be rewritten in plain English. The Returns section also needs to have a description added.

Reply:
@PipKat rewrote. Please help to check.